### ISSUE TITLE
Create option to enable/disable ServeUnknownFileTypes

### ISSUE THREAD

**Issue by MaximG1234** [nopSolutions/nopCommerce#4880]
nopCommerce version: 4.2, 4.3+

Steps to reproduce the problem: 

We require the ability to serve files with no extension because we are using LetsEncrypt Http authentication (https://letsencrypt.org/docs/challenge-types/) and the file created - that must be uploaded to the server - does not contain a file extension.

Typically the path will look something like this:

`http://nop43.com/.well-known/acme-challenge/mGCczU2GeRLcMKnxwFx0v64K3JRx-TgsxAzQo_s5ayk`

Unfortunately NopCommerce (understandably) doesn't not set the ServeUnknownFileTypes value in ApplicationBuilderExtensions and does not appear to provide a way to override this option. 

I have been able to work around the issue by creating a plugin that implements `INopStartup` and does 

```
        public void Configure(IApplicationBuilder application)
        {
            application.UseStaticFiles(new StaticFileOptions { ServeUnknownFileTypes = true });
        }
```
but I am not 100% sure if this doesn't break anything (is application.UseStaticFiles() cumulative, I believe it is?), further it would be nice to be able to temporarily toggle this value ideally without the use of a plugin.

**Comment by AndreiMaz** [comment]
Let's make it configurable.

**Comment by MaximG1234** [comment]
Thanks @AndreiMaz, that would be amazing.

**Comment by exileDev** [comment]
Closed #4880
